src/Elastica/Client.php

```diff
@@ -50,6 +51,16 @@ public function request($path, $method = Request::GET, $data = [], array $query
         $response = parent::request($path, $method, $data, $query, $contentType);
         $responseData = $response->getData();
 
+        $transportInfo = $response->getTransferInfo();
+        $connection = $this->getLastRequest()->getConnection();
+        $forbiddenHttpCodes = $connection->hasConfig('http_error_codes') ? $connection->getConfig('http_error_codes') : [];
+
+        if (isset($transportInfo['http_code']) && in_array($transportInfo['http_code'], $forbiddenHttpCodes, true)) {
+            $message = sprintf('Error in transportInfo: response code is %s, response body is %s', $transportInfo['http_code'], $responseData);
+            throw new ClientException($message);
+        }
+
+
```

Comment:
There are two empty line, one of them can be removed :)

Reply:
All resolved, sorry for this one hehehe